MAINTAINERS.md

```diff
@@ -1,11 +1,30 @@
 # Maintainers
 This file contains a list of the maintainers of tldr-pages.
 
+
+## Current collaborators
+The following people are collaborators on this repository, enabling them to 
+label and close issues, and review and merge pull requests. An automated list 
+can be found [here](https://github.com/tldr-pages/tldr/settings/collaboration).
+
+ - Max Xu ([@jsonbruce](https://github.com/jsonbruce))
+ - Jeef ([@jeeftor](https://github.com/jeeftor))
+ - Muhammad Falak R Wani ([@mfrw](https://github.com/mfrw))
+
+
+## Current members
+These people are members of the tldr-pages organization. An automated list can 
```

Comment:
Please also mention what they can/should do, similarly to what you wrote in the previous section. Let me know if this information is not clear.